projects/element-ng/typeahead/si-typeahead.component.html

```diff
@@ -46,4 +48,9 @@
       />
     </li>
   }
+  @if (createOption) {
+    <li class="dropdown-item" [siAutocompleteOption]="undefined">
```

Comment:
seems like accessibility is broken for whole `typeahead` itself.  voiceover on mac doesn't seem to work. May be a separate task to fix it.